services/brig/src/Brig/API/Client.hs

```diff
@@ -101,17 +124,33 @@ rmClient u con clt pw =
         _ -> Data.reauthenticate u pw !>> ClientDataError . ClientReAuthError
       lift $ execDelete u (Just con) client
 
-claimPrekey :: UserId -> ClientId -> AppIO (Maybe ClientPrekey)
-claimPrekey u c = do
+claimPrekey :: MappedOrLocalId Id.U -> ClientId -> AppIO (Maybe ClientPrekey)
+claimPrekey u c = case u of
+  Local localUser ->
+    claimLocalPrekey localUser c
+  Mapped _ ->
+    -- TODO: error?
```

Comment:
no not an error right? we contact the other backend and ask it to claim a prekey for us no?

Reply:
Only until we implement the logic. In Galley, we fail with some `federation-not-implemented` in these cases (a remote ID being used, but the function not being prepared for it).

Reply:
I removed all of these now. Still not 100% happy with it, but seems reasonable, since it will just behave the same as if a local user was not found.